client/state/reader/follows/reducer.js

```diff
@@ -82,6 +83,13 @@ export const items = createReducer( {}, {
 			[ urlKey ]: merge( {}, state[ urlKey ], { is_following: false } ),
 		};
 	},
+	[ READER_FOLLOW_ERROR ]: ( state, action ) => {
+		const urlKey = prepareComparableUrl( action.payload.url );
+		return {
+			...state,
+			[ urlKey ]: merge( {}, state[ urlKey ], { error: action.payload.error, is_following: false } ),
```

Comment:
> I don't think we should set is_following here, let the follow or unfollow action take care of it as necessary.

Fixed in da2ff7b 👍 

> do we also need a way to unset the error?

I'm trying to think of a scenario where we'd need to clear it out. Invalid feed becomes valid?